techniques/html/H28.html

```diff
@@ -124,21 +123,21 @@ <h3>Example 2: Using <code class="language-html">dfn</code> and <code class="lan
       </section>
 
       <section class="example" id="example-3-using-the-abbr-element-to-expand-an-acronym">
-         <h3>Example 3: Using the <code class="language-html">abbr</code> element to expand an acronym</h3>
+         <h3>事例 3: <code class="language-html">abbr</code> 要素を用いて頭字語の元の語を示す</h3>
          <pre xml:space="preserve"><code class="language-html">&lt;p&gt;Recent updates to the &lt;abbr title="Cascading Style Sheets"&gt;CSS&lt;/abbr&gt; color module ...&lt;/p&gt;</code></pre>
       </section>
 
       <section class="example" id="example-4-using-the-abbr-element-to-expand-an-initialism">
-         <h3>Example 4:  Using the <code class="language-html">abbr</code> element to expand an initialism</h3>
+         <h3>事例 4: <code class="language-html">abbr</code> 要素を用いて頭文字語の元の語を示す</h3>
```

Comment:
「解説」の冒頭で **expansions** or definitions for abbreviations を「略語の**展開形**又は定義」と訳を変更しているので (WCAG 2.1 達成方法集では「略語に対して**元の語**又は定義」と訳出していた)、それに合わせて「元の語」を「展開形」にしてみました。
```suggestion
         <h3>事例 4: <code class="language-html">abbr</code> 要素を用いて頭文字語の展開形を示す</h3>
```